src/components/HeaderProduct/HeaderProduct.tsx

```diff
@@ -9,27 +9,45 @@ export type ProductEntity = ProductSwitchItem;
 export type PartyEntity = PartySwitchItem;
 
 export type HeaderProductProps = {
-  productId?: string;
-  productsList: Array<ProductEntity>;
-  partyId?: string;
-  partyList?: Array<PartyEntity>;
-  onSelectedProduct?: (product: ProductSwitchItem) => void;
-  onSelectedParty?: (party: PartySwitchItem) => void;
+  borderBottom?: number;
+  borderColor?: string;
+  chipColor?:
+    | "default"
+    | "indigo"
+    | "primary"
+    | "secondary"
+    | "error"
+    | "info"
+    | "success"
+    | "warning";
```

Comment:
Is there a way to list all these color values without having to repeat each color value here?

Reply:
We can create and use a new type, like what has been done [here](https://github.com/pagopa/mui-italia/blob/49545b73fcdf34a9d532a0c5ce38c89f137d912c/src/components/Tag/Tag.tsx#L7)

Reply:
It's a small improvement for sure 👍🏻